src/util/generic-tests.jsx

```diff
@@ -7,7 +7,7 @@ import _ from 'lodash';
 import * as lucid from '../index';
 
 // Common tests for all our components
-export function common(Component, getDefaultProps=_.noop) {
+export function common(Component, { getDefaultProps = _.noop, selector = null } = {}) {
```

Comment:
I had to add a `selector` optional argument in case it's not possible to simply use the first element. This should help fix the common tests with `Banner` as well for #18 .

I moved `getDefaultProps` into an options object and gave it the same default value.